### DISCUSSION TITLE
Startls error from Java application

### DISCUSSION THREAD

**Discussion by hvisage** [postalserver/postal#2517] [Help with using Postal]
Good day,

 Trying to make a Java application (CrushFTP) talk to postal, but keep getting this error:

```
Server:smtp.xxxxx.yyyy
Port:25
User:xxx/xxxx-yyy
Error:javax.mail.MessagingException: Could not convert socket to TLS;
nested exception is:
javax.net.ssl.SSLException: Received fatal alert: internal_error

```

doing a snoop via the firewall, seeing the traffic from Postal that the TLS error comes from the POstal side:

<img width="1352" alt="Untitled" src="https://github.com/postalserver/postal/assets/11662665/2bf501fd-8038-4dc7-9390-13f9372745e2">

**Comment by sinnbeck** [comment]
How did you set up tls in postal? Using a certificate from Caddy?

**Reply by hevisko** [reply]
Using the "nginx" certbot certificates at present (might need to upgrade later to a caddy version)

**Comment by willpower232** [comment]
It would be worth looking at the smtp logs to see what the problem was with Postal, it may be that the certificate you provided didn't work properly.

**Comment by hevisko** [comment]
The problem, very... descriptive everywhere, seems that I had to enable TLSv1.3 in the CrushFTP SSL settings, and that solved the problem
